src/workloads/query/InWithVariedArraySize.yml

```diff
@@ -17,11 +17,10 @@ GlobalDefaults:
   Collection: &Collection Collection0
   DocumentCount: &DocumentCount 1e6
   MaxPhases: &MaxPhases 7
-  # This is internalQueryMaxScansToExplode - 3, since we append the generated array to [1, 2, 3] so
-  # we can actually match "int".
   internalQueryMaxScansToExplode: &internalQueryMaxScansToExplode 197
-  InUptoMaxScansToExplode: &InUptoMaxScansToExplode {$in: {^Concat: {arrays: [[1, 2, 3], {^Array: {of: {^Inc: {start: 4}}, number: {^Inc: {start: 0, end: *internalQueryMaxScansToExplode}}}}]}}}
-  InAboveMaxScansToExplode: &InAboveMaxScansToExplode {$in:  {^Concat: {arrays: [[1, 2, 3], {^Array: {of: {^Inc: {start: 4}}, number: {^Inc: {start: *internalQueryMaxScansToExplode, end: 2000}}}}]}}}
+  randomIntUptoDocCount: &randomIntUptoDocCount {^RandomInt: {min: 0, max: 1000000}}
```

Comment:
Could you please replace max: 1e6 with  max: *DocumentCount.

Reply:
Done.